docs/whats-new/dotnet-7-docs.md

```diff
@@ -0,0 +1,295 @@
+---
+title: ".NET docs: What's new for .NET 7"
+description: "What's new in the .NET docs for the .NET 7 release."
+ms.date: 11/08/2022
+---
+
+# .NET docs: What's new for the .NET 7 release
+
+Welcome to what's new in the .NET docs for the .NET 7 release. This article lists some of the major changes to docs related to .NET feature updates.
+
+## .NET breaking changes
+
+- [Breaking changes in .NET 7](../core/compatibility/7.0.md)
+
+## .NET fundamentals
+
+### New articles
+
+- [What's new in .NET 7](../core/whats-new/dotnet-7.md)
+- [Introduction to AOT warnings](../core/deploying/native-aot/fixing-warnings.md)
+- [Diagnostic monitoring and collection utility (dotnet-monitor)](../core/diagnostics/dotnet-monitor.md)
+- [Containerize a .NET app with dotnet publish](../core/docker/publish-as-container.md)
+- [System.Threading.Channels library](../core/extensions/channels.md)
+- [Rate limit an HTTP handler in .NET](../core/extensions/http-ratelimiter.md)
+- [Upgrade a WCF Server-side Project to use CoreWCF on .NET 6](../core/porting/upgrade-assistant-wcf.md)
+- [dotnet workload command](../core/tools/dotnet-workload.md)
+- [NuGet signed package verification](../core/tools/nuget-signed-package-verification.md)
+- [.NET SDK error list](../core/tools/sdk-errors/index.md)
+- [SYSLIB diagnostics for regex source generation](../fundamentals/syslib-diagnostics/syslib1040-1049.md)
+- [SYSLIB diagnostics for p/invoke source generation](../fundamentals/syslib-diagnostics/syslib1050-1069.md)
+- [SYSLIB diagnostics for JavaScript interop source generation](../fundamentals/syslib-diagnostics/syslib1070-1089.md)
+- [SYSLIB0038: SerializationFormat.Binary is obsolete](../fundamentals/syslib-diagnostics/syslib0038.md)
+- [SYSLIB0039: SslProtocols.Tls and SslProtocols.Tls11 are obsolete](../fundamentals/syslib-diagnostics/syslib0039.md)
+- [SYSLIB0040: EncryptionPolicy.NoEncryption and EncryptionPolicy.AllowNoEncryption are obsolete](../fundamentals/syslib-diagnostics/syslib0040.md)
+- [SYSLIB0041: Some Rfc2898DeriveBytes constructors are obsolete](../fundamentals/syslib-diagnostics/syslib0041.md)
+- [SYSLIB0042: FromXmlString and ToXmlString on ECC types are obsolete](../fundamentals/syslib-diagnostics/syslib0042.md)
+- [SYSLIB0043: ECDiffieHellmanPublicKey.ToByteArray is obsolete](../fundamentals/syslib-diagnostics/syslib0043.md)
+- [SYSLIB0044: AssemblyName.CodeBase and AssemblyName.EscapedCodeBase are obsolete](../fundamentals/syslib-diagnostics/syslib0044.md)
+- [SYSLIB0046: ControlledExecution.Run should not be used](../fundamentals/syslib-diagnostics/syslib0046.md)
+- [SYSLIB0047: XmlSecureResolver is obsolete](../fundamentals/syslib-diagnostics/syslib0047.md)
```

Comment:
Is there an overview for this section? Would that simplify the output?

Reply:
There is an overview, but it includes more than just what's new for .NET 7, so I think this is correct as is.